server/download.go

```diff
@@ -171,8 +171,9 @@ func (b *blobDownload) Prepare(ctx context.Context, requestURL *url.URL, opts *r
 			offset += size
 		}
 	}
-
-	slog.Info(fmt.Sprintf("downloading %s in %d %s part(s)", b.Digest[7:19], len(b.Parts), format.HumanBytes(b.Parts[0].Size)))
+	if len(b.Parts) != 0 {
```

Comment:
maybe just use `else` ?

the condition on line 144 does `if len(b.Parts) == 0 {`